bitnami/README.md

```diff
@@ -45,3 +47,12 @@ pubsub
 * add more resources to be base if needed
 * PR the changes
 * use the new base in `kubernetes-manifests`
+
+## Monitoring
+
+For clusters using these manfiests, there are Grafana dashboards available for:
+
+  - [A general
+    overview](https://grafana.dev.aws.uw.systems/d/qoF35R8Vk/bitnami-kafka-overview?from=now-24h&to=now&var-datasource=default&var-namespace=pubsub&var-topic=All&var-consumergroup=All&orgId=1&refresh=1m)
+  - [Topic and consumer group
```

Comment:
This one has preselected the es-topic-indexer. I'd leave it to all.

Reply:
> This one has preselected the es-topic-indexer. I'd leave it to all or just take out any parameters from the link

Good spot: [7de2089](https://github.com/utilitywarehouse/kafka-manifests/pull/59/commits/7de20896d5cec8b96e257c343422a4e7b845fa92)